src/pydisagg/ihme/preprocess.py

```diff
@@ -0,0 +1,417 @@
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+from typing import Dict, List
```

Comment:
`Tuple, Dict, List` will be deprecated from `typing` module starting Oct 2025, might be good to switch to directly using `tuple, dict, list`.
https://peps.python.org/pep-0585/#implementation